lib/tasks/migrate_fields_to_rich_text.rake

```diff
@@ -0,0 +1,25 @@
+namespace :standup_meeting do
+  desc 'Migrate standup meeting data to action text'
+
+  task migrate_data_to_action_text: :environment do
+    failed_updates = []
+
+    StandupMeeting.where(status: 'completed').find_each do |meeting|
+      meeting.update!(
+        today_work_description: meeting.read_attribute_before_type_cast(:today_work_description),
+        yesterday_work_description: meeting.read_attribute_before_type_cast(:yesterday_work_description),
+        blockers_description: meeting.read_attribute_before_type_cast(:blockers_description)
+      )
```

Comment:
i'm kind of confused as to what this exactly does?

Reply:
This assigns the information that's currently in the `today_work_description`, `yesterday_work_description`, and `blockers_description` columns and assigns them as new `action_rich_texts` records associated with the standup meeting.

The new `has_rich_text` associations are set with the same name as the old columns. I also didn't expect this to work, but I just ran it locally and it does lol. I guess there's something funky with the way the reads/writes work regarding rich text associations having the same name as a column.

I probably would've done something like
- Introduce rich text associations with a different name than the current columns
- Move old column's information over to rich text
- Remove old columns (this should be done anyway, albeit in a separate PR from this one of course)
- If I wanted the rich text association names to be the same as the old column names, write a new task that does `ActionText::RichText.where(record_type: "StandupMeeting")` to pull the standup rich text records and then change their `name` attributes to match the old column names. And then make that adjustment in the standup meeting model file

That's all more complicated than what Gustavo has here, but also makes more sense to me lol.

Reply:
> This assigns the information that's currently in the `today_work_description`, `yesterday_work_description`, and `blockers_description` columns and assigns them as new `action_rich_texts` records associated with the standup meeting.
> 
> The new `has_rich_text` associations are set with the same name as the old columns. I also didn't expect this to work, but I just ran it locally and it does lol. I guess there's something funky with the way the reads/writes work regarding rich text associations having the same name as a column.
> 
> I probably would've done something like
> 
> * Introduce rich text associations with a different name than the current columns
> * Move old columns' information over to rich text
> * Remove old columns (this should be done anyway, albeit in a separate PR from this one of course)
> * If I wanted the rich text association names to be the same as the old column names, write a new task that does `ActionText::RichText.where(record_type: "StandupMeeting")` to pull the standup rich text records and then change their `name` attributes to match the old column names. And then make that adjustment in the standup meeting model file to the `has_rich_text` associations.
> 
> That's all more complicated than what Gustavo has here, but also makes more sense to me lol.

What you just explained above is exactly how I would have done it also, but if this works, thats fine by me lol

Reply:
Sorry @toyhammered, didn't get a chance to address your question. But Thanks @JoshDevHub for getting to it! 
Also, forgot to submit a chained pr with the migration to remove the old fields. I'm just going to wait until this one is merged!